setup.py

```diff
@@ -58,6 +58,11 @@ def read_file(filename, encoding='utf8'):
 else:
     install_requires.append('mock')
 
+if sys.version_info < (2, 7, 9):
```

Comment:
I opened https://github.com/letsencrypt/letsencrypt/issues/1997 to address this.